translations/nl.po

```diff
@@ -631,22 +627,18 @@ msgid "Paste XML"
 msgstr "XML Plakken"
 
 #: src/ui_parts/settings_menu.gd
-#, fuzzy
 msgid "New palette"
 msgstr "Nieuw element"
 
 #: src/ui_parts/settings_menu.gd
-#, fuzzy
 msgid "New palette from XML"
 msgstr "Nieuw element"
```

Comment:
It seems like these had their fuzzy marker removed, even though they weren't correct translations.